.changeset/tasty-spies-guess.md

```diff
@@ -0,0 +1,6 @@
+---
+'website': major
```

Comment:
Updated... just to double check, `website` part has a change. It's a documentation for new component. Definitely not a major change, since it's new feature/not breaking existing solution, but shouldn't it stay `minor` here as well?

Reply:
Yes the website has changes in the sense of git changes, but in terms of the deployed release and semantic versioning of what triggers npm package registry updates, we don't want to include the `website` package as it's not a part of the packaged and released code. 

Remove the line of `'website': major` and change `'@project44-manifest/react'` to `minor` and then this release file will be good to go